packages/toggle/src/Toggle.tsx

```diff
@@ -370,30 +370,34 @@ function Toggle({
   disabled = false,
   onChange: onChangeProp,
   onClick: onClickProp,
-  checked: checkedProp,
+  checked: controlledChecked,
   ...rest
 }: ToggleProps) {
   validateAriaLabelProps(rest, Toggle.displayName);
 
   const [buttonElement, setButtonElement] = useState<HTMLButtonElement | null>(
     null,
   );
-  const [checked, setChecked] = useState(checkedProp ?? false);
-  const normalizedChecked = checkedProp ?? checked;
+  const [checked, setChecked] = useState(false);
+
+  const isControlled = typeof controlledChecked === 'boolean' ? true : false;
+  const normalizedChecked = controlledChecked ?? checked;
```

Comment:
where is this used?

Reply:
normalizedChecked is used to determine the aria-checked value. I believe this was in the code before, I don't think I added it.

the isControlled check is used in the clickHandler